packages/web-components/@sfx/products/stories/index.ts

```diff
@@ -0,0 +1,78 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, number } from '@storybook/addon-knobs';
+
+import '../src/index.ts';
+import { ProductModel } from '../../product/src/product';
```

Comment:
Should be imported from the package
```suggestion
import { ProductModel } from '@sfx/product';
```